src/components/LicenseRevokeModal/index.jsx

```diff
@@ -47,18 +48,20 @@ class LicenseRevokeModal extends React.Component {
     const {
       user,
       sendLicenseRevoke,
+      fetchSubscriptionDetails,
+      fetchSubscriptionUsers,
+      searchQuery,
     } = this.props;
-
-    const options = {};
-    options.assignments = [{
-      uuid: user.uuid,
-      email: user.emailAddress,
-      licenseStatus: user.licenseStatus,
-    }];
-
+    const options = { userId: user.userId };
     return sendLicenseRevoke(options)
-      .then((response) => {
-        this.props.onSuccess(response);
+      .then(async (response) => {
+        try {
+          await fetchSubscriptionDetails();
```

Comment:
After revoking a license, the number of allocated licenses will decrease by 1, so we need to refresh the "X of Y licenses allocated data. Similarly, the counts in the tab navigation and the data for the table will now have changed, so we need to refresh that, too.